src/database/migrations/2022_01_10_99999_create_chatify_messages_table.php

```diff
@@ -16,7 +16,7 @@ public function up()
         Schema::create('ch_messages', function (Blueprint $table) {
             $table->uuid('id')->primary();
             $table->bigInteger('from_id');
-            $table->bigInteger('to_id');
+	        $table->string('to_channel_id')->nullable();
             $table->string('body',5000)->nullable();
             $table->string('attachment')->nullable();
             $table->boolean('seen')->default(false);
```

Comment:
Hi, "seen" not work as boolean. I have changed to string because functions makeSeenm and countUnseenMessages use jsoncontain with this field....